lib/cog/command/command_resolver.ex

```diff
@@ -0,0 +1,56 @@
+defmodule Cog.Command.CommandResolver do
+
+  alias Cog.Repo
+  alias Cog.Queries.Command
+  alias Cog.Queries.Alias
+  alias Piper.Command.SemanticError
+
+  def find_bundle(name) when is_binary(name) do
+    if String.contains?(name, ":") do
+      :identity
+    else
+      case Repo.all(Command.bundle_for(name)) do
```

Comment:
Since bundle names trump alias names do we prevent users from creating aliases which conflict with installed bundles? What happens when a user creates an alias and later a bundle with a conflicting name is installed?